browser-test/src/trusted_intermediary.test.ts

```diff
@@ -493,39 +511,13 @@ test.describe('Trusted intermediaries', () => {
     await tiDashboard.searchByNameAndDateOfBirth('first1', '', '', '2021')
     await waitForPageJsLoad(page)
 
+    await expect(
+      page.getByRole('heading', {name: 'Displaying 1 client'}),
+    ).toBeVisible()
     await tiDashboard.expectDashboardContainClient(client1)
     await tiDashboard.expectDashboardNotContainClient(client2)
   })
 
-  test('empty search parameters returns all clients', async () => {
```

Comment:
Thanks for adding this button to clear search. Question: What is the expected behavior when search with empty search parameters?

Reply:
The expected behavior is to display all clients in an unfiltered list.

Reply:
I removed this test because I added that test scenario in the 'search For Client In TI Dashboard' test (see line 442).  I consolidated the two tests for better test performance, as Gwen suggested.

Reply:
I see. makes sense. thanks.